app/src/main/java/de/stephanlindauer/criticalmaps/Main.java

```diff
@@ -56,6 +70,15 @@ public void onCreate(Bundle bundle) {
         locationUpdatesService.initialize(getApplication());
 
         startSyncService();
+
+        drawerNavigation.setNavigationItemSelectedListener(this);
+
+        setSupportActionBar(toolbar);
+        ActionBarDrawerToggle mDrawerToggle = new ActionBarDrawerToggle(this, drawerLayout, toolbar, R.string.open, R.string.close);
```

Comment:
those strings are for accessibility so especially the close string makes no sense here. I suggest something like "open drawer" and "close drawer"